apps/web/screens/Project/Project.tsx

```diff
@@ -0,0 +1,212 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { ReactElement } from 'react'
+import { withMainLayout } from '@island.is/web/layouts/main'
+import {
+  ContentLanguage,
+  GetNewsQuery,
+  Query,
+  QueryGetNamespaceArgs,
+  QueryGetProjectPageArgs,
+} from '@island.is/web/graphql/schema'
+import { GET_NAMESPACE_QUERY, GET_NEWS_QUERY } from '../queries'
+import { Screen } from '../../types'
+import { useNamespace } from '@island.is/web/hooks'
+import { CustomNextError } from '@island.is/web/units/errors'
+import useContentfulId from '@island.is/web/hooks/useContentfulId'
+import { GET_PROJECT_PAGE_QUERY } from '@island.is/web/screens/queries/Project'
+import {
+  DefaultProjectHeader,
+  LatestNewsSectionSlider,
+  OrganizationSlice,
+  Section,
+} from '@island.is/web/components'
+import Head from 'next/head'
+import {
+  GridColumn,
+  GridContainer,
+  GridRow,
+  Navigation,
+  Text,
+} from '@island.is/island-ui/core'
+import { richText, SliceType } from '@island.is/island-ui/contentful'
+import { QueryGetNewsArgs } from '@island.is/api/schema'
+import SidebarLayout from '@island.is/web/screens/Layouts/SidebarLayout'
+import NextLink from 'next/link'
+import { useRouter } from 'next/router'
+
+interface ProjectWrapperProps {
+  withSidebar?: boolean
+  sidebarContent?: ReactElement
+}
+
+const ProjectWrapper: React.FC<ProjectWrapperProps> = ({
+  withSidebar = false,
+  sidebarContent,
+  children,
+}) => {
+  return withSidebar ? (
+    <SidebarLayout isSticky={true} sidebarContent={sidebarContent}>
+      {children}
+    </SidebarLayout>
+  ) : (
+    <GridContainer>
+      <GridRow>
+        <GridColumn
+          paddingTop={6}
+          paddingBottom={6}
+          span={['12/12', '12/12', '10/12']}
+          offset={['0', '0', '1/12']}
+        >
+          {children}
+        </GridColumn>
+      </GridRow>
+    </GridContainer>
+  )
+}
+
+interface PageProps {
+  projectPage: Query['getProjectPage']
+  news: GetNewsQuery['getNews']['items']
+  namespace: Query['getNamespace']
+}
+
+const ProjectPage: Screen<PageProps> = ({ projectPage, news, namespace }) => {
+  const n = useNamespace(namespace)
+  const router = useRouter()
+  useContentfulId(projectPage.id)
+
+  const subpage = projectPage.projectSubpages.find((x) => {
+    return x.slug === router.query.subSlug
+  })
+
+  return (
+    <>
+      <Head>
+        <title>{projectPage.title} | Ísland.is</title>
+      </Head>
+      <DefaultProjectHeader projectPage={projectPage} />
+      <ProjectWrapper
+        withSidebar={projectPage.sidebar}
+        sidebarContent={
+          <>
+            <Navigation
+              baseId="pageNav"
+              items={projectPage.sidebarLinks?.map((x) => ({
```

Comment:
it's usually cleaner to destructure the variable rather than using a placeholder name if possible, eg.
```
.map(({ text, url }) => ({
```

ps. this may be a copy paste of code that was already in place, in which case, sorry about that

Reply:
Good advice, changed it.